monkestation/code/modules/art_sci_overrides/artifact_testers/analysis_form.dm

```diff
@@ -96,10 +97,10 @@
 			continue //shhhhh
 		artifact_names += initial(subtype.type_name)
 
-	.["allorigins"] = origins_names
-	.["allfaults"] = allfaults
-	.["alltypes"] = artifact_names
-	.["alltriggers"] = trigger_names
+	.["allorigins"] = sort_list(origins_names)
+	.["allfaults"] = sort_list(allfaults)
+	.["alltypes"] = sort_list(artifact_names)
+	.["alltriggers"] = sort_list(trigger_names)
```

Comment:
Use binary insert macros to pre-sort these. `sortTim`, and by extension `sort_list`, is not exactly very performance-friendly.